plugins/helper.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * Set default start time and end time
+ * @method setDefaultStartEnd
+ * @param  {String}         start     start time
+ * @param  {String}         end       end time
+ * @param  {Number}         maxDay    max day range
+ * @return {Object}                   Default start time and end time
+ */
+function setDefaultTimeRange(start, end, maxDay) {
+    const endTime = end || new Date(Date.now()).toISOString();
+    const startTime = start || new Date(new Date().setDate(new Date(endTime).getDate() - maxDay))
```

Comment:
I would recommend using `moment.js` for this. This can be concisely expressed using https://momentjs.com/docs/#/manipulating/subtract/

Reply:
I saw `moment.js` but not sure if we want to pull a new dependency since the calculation is quite simple.
I can do it if that's what we prefer.

Reply:
we are creating 6 date objects in this one function call. `moment` will abstract all this logic. and is well tested.

Reply:
@DekusDenial @jithin1987   you two can fight :p

Reply:
I agree on the performance aspect. If we are to replace with some alternative libraries from the link @DekusDenial posted, I am okay with it.

However some of the concerns are not applicable. This does not go into browser bundle, but on API server.

Reply:
I prefer to not pull in another dependency since the calculation isn't overly complicated. 
But I can switch to `dayjs` or `date-fns`, it looks more lightweight.

Reply:
1) no time zone exchange here
2) it’s ISO, u want one with faster parsing. This again brings up my question about why almost all our times are in ISO and not epoch which is more of the norm and no brainer in parsing

Reply:
Using `moment` blindly is like importing the whole `lodash/underscore` but u only use one function from it.